### PR TITLE
Adds security jumpskirts this time with less broken github

### PR DESCRIPTION
**Pull request by Qbopper**
:cl: Ausops and Qbopper
add: Added security jumpskirts - snag one from a security wardrobe locker.
/:cl:

i don't know if i was supposed to add my name to the cl or not but i can remove it if need be

I actually went in game and made sure they spawn and work and everything but I haven't done any git shit in a while so let's hope nothing goes wrong

image stolen from the last pr
skirt

edit: i wish you guys would come out of the woodwork to fight over shit that actually matters as much as you do with shit like female oriented clothing options

### CODE REVIEW

**Comment by Alexch2** [comment]
third time's the charm 👌

**Comment by Spaceinaba** [comment]
the sec skirt redemption arc begins

File: code/modules/clothing/under/jobs/security.dm [hunk 1]
```diff
@@ -28,6 +28,14 @@
...
 	item_color = "security"
 
+/obj/item/clothing/under/rank/security/skirt
+	name = "security jumpskirt"
+	desc = "A \"tactical\" security jumpsuit with the legs replaced by a skirt."
+	icon_state = "secskirt"
+	item_state = "r_suit"
+	item_color = "secskirt"
+	can_adjust = FALSE //you know now that i think of it if you adjust the skirt and the sprite disappears isn't that just like flashing everyone
+
```
Comment (GuppyLaxx) [inline code/modules/clothing/under/jobs/security.dm]: muh empty lines

**Comment by vuonojenmustaturska** [comment]
why do these have pockets

**Comment by Qbopper** [comment]
the top half i guess

**Comment by sawrge** [comment]
fetish content

**Comment by Arianya** [comment]
Unrelated this PR, but at some point we might want to get a version of the autodrobe that's in security to avoid having the lockers full of outfit bloat

**Comment by Qbopper** [comment]
The wardrobe lockers are separate from the actual sec equipment lockers, but that's a good idea and a better solution

I just cranked this out in 10 minutes because I was asked to

**Comment by Armhulen** [comment]
I don't believe that it fits with the unified theme of security and their job of "private hired security"

**Comment by Armhulen** [comment]
Also lets talk about that guppylax dude who pr'd this before you to beat you at it but then complained that he didn't know at all how to code or use git

**Comment by Shadowflame909** [comment]
I remember watching a cartoon one time where a skirt turned into full ninja battle armor. I think that's cool. I think you not naming this a kilt makes this a "top ten most controversial prs of all time." Lastly, as a security officer who has to take down threats. Can you tell me a skirt would not look out of place? It'd look more niormal on the shitpost that is hippiestation but it seems bizzare here.

**Comment by animetiddies** [comment]
this is a game where clowns beat the shit out of people all the time. what the fuck do you mean a skirt is "out of place"

**Comment by RogueSteampunker** [comment]
>skirt
>out of place

Riiiiggghhtt, don't want a fucking skirt breaking my immersion as a wizard turns into a rod to shove it up my ass, deal with multiple ACTUALLY CORRECT cults, and fucking ninjas from space. 

But sure, the fucking colored skirt is apparently not fitting the theme, despite the fact that we have fucking clowns. Just saying, I doubt NT in-universe would give a flying fuck about a single, non-intrusive skirt, and then immediately turn around an hire a clown for a space station.

So please, if you're argument has anything to do with the nonexistent slippery slope, or how muh immursion is broken by sec having them, please fuck off.

**Comment by GuppyLaxx** [comment]
@Armhulen you do know the original pr worked until a one line change which then the bot started complaining about lol

**Comment by GuppyLaxx** [comment]
`>`pr compiles
`>`make one line change that doesn't touch the .dmi
`>`bot complains about .dmi

**Comment by Shadowflame909** [comment]
Please don't strawman my argument, cat-meme player. I honestly have nothing against catgirls, who cares what race they play as, just let people enjoy the game. Yet, I can't agree with this shitpost of a PR. The argument that I'm getting at is "lol everything is so zany who cares if security valid-hunts in skirts" I don't think this is a very good argument considering how it's been said over and over again how we want to be a medium roleplay server. Firstly, Wizards being unstoppable reality warping juggernauts is there gimmick, I don't see how that's an argument. Also, "we have clowns" isn't a very good argument either. The clowns are not security personnel who have to stop and detain or kill threats on a daily basis. 

So, all in all, I believe this PR will bring us down to an even lower level of seriousness and roleplay that we're at currently. It doesn't give me any reasons for why security should be valid-hunting and arresting people in skirts. I'm taking away from this that "Lol omfg nyaa~ I'm valid-hunting in a skirt and I'm diverting everyone's attention by how ridiculous I'm being lol" The clown slips people and does silly shit. That's his job, and since it seems to be such a prominent argument to why security should run around and make normal things look otherwise ridiculous for such a prominently serious job. (To keep the peace..) I guess we should hold ourselves to a low-rp standard now in favor of this PR, I guess.

**Comment by animetiddies** [comment]
@Shadowflame909 lol

**Comment by Shadowflame909** [comment]
That's not an argument dood.

**Comment by animetiddies** [comment]
this is a pr about adding content not your personal debate club. you dont have a right to force anyone listen to your outbursts

**Comment by DaxDupont** [comment]
slippery slope

**Comment by Shadowflame909** [comment]
>The HOS already has a memeskirt

Whatever speed-merge this is why sybils better then bagil ree.

**Comment by RogueSteampunker** [comment]
@Shadowflame909 
>Medium RP
>M E D I U M  R P

Gimmie a moment. I gotta go kek a kek I've never keked before.

Oh, and here's the thing, bud. If a single fucking skirt is apparently going to bring us down on the idea of RP, then it was doomed from the start. Jusssttt saying.

**Comment by Shadowflame909** [comment]
I think it looks out of place and would fit more on hippie's security team then over here, that's all. But I digress.

**Comment by sawrge** [comment]
1) why would corporate security have different outfits when they should be easily recognizable even at a glance, IC or OOC
2) loose clothes in space/lo grav environments 
3) would offer less protection than a jumpsuit since more skin is exposed 

there's three reasonable ways this is immersion breaking

**Comment by sawrge** [comment]
of course there's plenty of other items in the game already guilty of any or all of the above and while i don't really agree two wrongs make a right thirty wrongs means there isn't much right left so the immersion argument is kind of moot anyway. 

kor said he's merging it at 24 hours and im not bothered enough by this to argue any more but it is further down the slippery slope at any rate

**Comment by Qbopper** [comment]
i don't give a shit and made this pr because i was asked to and honestly i wish people would argue as passionately as they do with shit like this on more important stuff

**Comment by pigeonsk** [comment]
To address point 1, the color scheme is the most important aspect of security recognizablity, which is preserved in this sprite.

**Comment by Armhulen** [comment]
@roguesteampunker

the themes of the game are wacky and the scenarios are wild but the universe it's in is grounded. the syndicate are still a terrorist organization that sends agents, jobs have uniforms that fit their role and they have jobs. what makes the game wacky are the players that spice rounds up, that punch people as clowns, that rod form up people's ass as you said. sure, the cult is actually real but it's not a silly theme, it's an actual cult that kidnaps people and they actually inscribe runes into their body.

I don't think my opinion is "please fuck off" worthy because i want the game to be thematic for the setting of the game. in any of the insane scenarios the game takes itself seriously. I just personally feel that a privately hired security team from an evil megacorporation would not have skirts on their security force

"But what about the hos skirt"
Thematically makes sense. Heads are representative of their department, and so it makes sense that most of them have more casual gear sometimes. The captain has a parade uniform and the hop has the formal attire, and it so happens the hos does too! I like it.

"But what about warden gear"
If the warden has a skirt, I don't like it. The warden seemingly has mostly in theme gear for their job.

**Comment by Armhulen** [comment]
basically sawrge's points, but expanded so he doesn't have to reply to the thread

**Comment by PKPenguin321** [comment]
this is one of those things that is not inherently funny in any way. like the ugandan knuckles meme, this is only funny because your peers tell you it is, and in a week it will no longer be funny and you will no longer have any investment in it being in the game.

if the robby rotten suit didnt make it into the clown vendor i dont see why this should. flavor of the week meme trash

**Comment by pigeonsk** [comment]
Ugadan Knuckles wasn't funny because it was inherently unfunny, it became unfunny after ~normies~ people abused it over and over again repeatedly, as well as spreading to people who had no understanding nor context of the meme who then bandwagoned on it as well, resulting in one of the fastest and biggest meme deaths I've ever seen. That example does not apply here.

**Comment by PKPenguin321** [comment]
exactly. this is only going to be funny for a week or two tops during which it will be ran into the ground, and after which the only people who will wear it are people who lack a shred of irony like miku frost (and people who don't realize how horribly unfunny they are).

that example absolutely applies here. there is nothing inherently funny or good about this. it's a sec officer, but theyre wearing a skirt instead of a regular uniform! #wow #woah #lmfao

**Comment by pigeonsk** [comment]
Where are you getting the notion that the primary usage of this uniform is a meme/joke

It's literally just an alternative uniform like the nursesuit.

**Comment by PKPenguin321** [comment]
where aren't you getting that notion

**Comment by Armhulen** [comment]
i think he's actually saying it won't die out because we aren't normies

whhhhhhhhhhhhhhhhhhhhhhhhhhhhaaaaaaat

**Comment by Qbopper** [comment]
have you considered that some people would genuinely like to wear this as an article of clothing and they do not consider it an epic xd meme

i can think of at least 3 people off the top of my head and i don't even play

anyways: kor told me that ausops wanted me to put an alt armor in the lockers with this pr to keep things even but it turns out that what i was asked to do is how the armor is distributed currently so thank you for that kor and ausops

**Comment by pigeonsk** [comment]
Nursesuit, and the many other feminine versions of uniforms that are already in the game.

**Comment by pigeonsk** [comment]
@Armhulen That is not what I'm saying.

**Comment by Armhulen** [comment]
@pigeonsk doctors are civilians, i don't see why they can't wear skirts

**Comment by KorPhaeron** [comment]
"Are Security Skirts Immersive" - the greatest thread in the history of github, locked by a moderator after 12,239 pages of heated debate,

**Comment by PKPenguin321** [comment]
>i can think of at least 3 people off the top of my head and i don't even play

yes, such as our pal miku frost and Donut S. Teel as i mentioned earlier

**Comment by Qbopper** [comment]
i can see that this is going to be a bad faith argument so i will back out now and leave off with this

if it makes people happy and has no direct impact on anything beyond that, why is it a problem for it to be merged

**Comment by Shadowflame909** [comment]
Because other people doubt what the intentions for this are. I'm not sure about this bop.

**Comment by Armhulen** [comment]
@Shadowflame909 ?????????

@Qbopper 
because it's not in theme of the game. civilian jobs can and should get skirts so players can build characters how they want but it makes no sense for security to have them

**Comment by Shadowflame909** [comment]
Armhulenn, we've said our peace. But the (*insert catgirl player base) wildly supports this. I don't think any more viewpoints will change if they haven't already honestly. It's just a piece of clothing. We won't see ERP get merged, right?

**Comment by Arianya** [comment]
It will always amuse me that /tg/ manages to have more hotly contested conversations about the adding of a cosmetic feature then they will about the implementation of complex, dangerous systems like Fusion which impact the game at large.

**Comment by PKPenguin321** [comment]
>i can see that this is going to be a bad faith argument

what did he mean by this

**Comment by Shadowflame909** [comment]
@Arianya  Most of us don't know how to use atmos/ don't play ai.

**Comment by Anonmare** [comment]
The amount of argumentation this concept has caused has convinced me that our playerbase are all universally suffering from some form of spectrum disorder

**Comment by 81Denton** [comment]
Where do you think we are.png

**Comment by pigeonsk** [comment]
https://en.wikipedia.org/wiki/Pteruges
https://en.wikipedia.org/wiki/History_of_the_kilt
Skirts have been historically and successfully used in combat.

This is also the future so there is a lot of leeway here for it to make sense, I don't understand why you all are nitpicking so much about fashion when our atmos sim code still says space is cold.

**Comment by Arianya** [comment]
@Shadowflame909 the vast majority of us will also have no interaction with the skirt other then observing a small collection of pixels on someone elses mob.

**Comment by Arianya** [comment]
Anyway to provide actual counter-arguments:

**Tone:** Tone has long been dead in ss13 and to claim secskirts of all things are somehow dangerous to the tone is really weird and seems almost like a red herring??

**ERPers:** Multiple server votes have proven that the majority of the playerbase accepts and likes features traditionally identified as being "ERPer" features such as catpeople. Additionally fashion choices tend to appeal to a broad spectrum of players, and have useful gameplay effects since they can result in an item being more highly desired in spite of no mechanical advantage.

**"Intentions"/Other servers would use it** This seems kinda a weird, unfounded argument based off boogeymanning it so I don't really know how to respond to this other then asking for clarification

**Comment by RogueSteampunker** [comment]
@Armhulen 

Arm, can we just talk for a moment about how you seem to assume that, since the apparent inuniverse is arguably somewhat sane, Security wearing skirts is suddenly too much? Cause like I said in the discord, a skirt apparently being enough to break the theme of the game, then either you don't understand it as well as you say, or something else completely unrelated to skirts is fucked up.

And once again, lemme clarify: This is the same station that found it reasonable to hire in fucking clowns and mimes for a full-grown crew. The same station that is completely and utterly allowed to create weapons of mass-destruction and utter chaos, and all of this is ran by a corporation that couldn't give less of a care about the people who work there, as long as they're still on top. So the fact that skirts are apparently a fucking serious concern that they'd care about, even when it's been shown that working and going into combat with such has worked fine before, seems just a little bit farfetched.

Jussssttt sayin.

**Comment by Shadowflame909** [comment]
I have no more counter-arguments. I shall cease this purist mindset, but if this gets merged and I'm seeing security in dorms 24/7. I'm calling goof.

**Comment by RogueSteampunker** [comment]
@Shadowflame909 

>but if this gets merged and I'm seeing security in dorms 24/7. I'm calling goof.

You're joking, right? You don't actually think people run into the dorms to fuck on TG, right?

**Comment by KorPhaeron** [comment]
@RogueSteampunker Hiring entertainers for a long term space station makes sense and so does weapons research

**Comment by Alexch2** [comment]
>argues for 4 hours
>Haha, I'm joking!
🤔 🤔 🤔

**Comment by Shadowflame909** [comment]
First of all, I am no longer salty shadow. My salt high died an hour ago, some of my points have been refuted. I have run out of steam to argue against these points. Now I'm just shitposting. So eat it Alex!

**Comment by RogueSteampunker** [comment]
@KorPhaeron Right, but the point I'm getting at is that it's clear that NT is basically just doing the basic necessities to keep the stations afloat, and being able to claim that they're doing the proper steps to a good crew. Yeah, weapon research and entertainers would be crucial to keep the crew protected and happy. And yet, stuff like the BSA or the meteor shields are locked behind multiple steps of work when it would be reasonably protected from the start by anyone less dastardly than NT, and the entertainment is a clown and a mime, who're even cited inuniverse as being pretty shit at their jobs.

I'm just trying to look at it from the point of view of NT just being more lazy than malicious, and with how easy it seems for them to be able to crank out new stations and clone workers again and again, it would be fitting.

**Comment by AdamElTablawy** [comment]
it's not a matter of theme for christ sake why is everyone shouting 'LMAO WE HAVE CLOWNS HOW ARE SKIRTS BREAKING IMMERSION'

it's that it's fucking stupid

that's it

it's stupid, it looks stupid, it sounds stupid (sec officer in a skirt lmao) and it's just going to cater to the kinds of obnoxious people who unironically wear this kind of shit a la miku frost

this stuff goes in the costume vendor or the clothes vendor at best, if you really insist on adding shit that fat acne-riddled neckbeards will use to frill up their 'female' characters, as opposed to the rest of us fat acne-riddled neckbeards who don't do that

**Comment by sawrge** [comment]
I know Kor mentioned his plan to merge this because it's inconsequential given how many clothes options are in the game already although sec is (for the most part) still pretty homogenized in terms of their uniforms. Formals aren't something always readily available and at the very least have the same outline, so I think there's still an argument to be made to keep sec looking uniform and standardized.

**Comment by Armhulen** [comment]
NT is as evil as the Syndicate

image

**Comment by Shadowflame909** [comment]
How come both sides just don't compromise and make it rare. Like, make only 1 and make it the insuls of security. Lore!

**Comment by RogueSteampunker** [comment]
@Armhulen Right, but if NT was even half as innovative and motivated as the syndicate, they wouldn't need to steal syndie gear for research :^)

**Comment by Armhulen** [comment]
@RogueSteampunker wokeness itt

**Comment by RogueSteampunker** [comment]
@Armhulen I mean, you're not wrong :p

**Comment by Wilchenx** [comment]
@AdamElTablawy you don't even play here anymore afaik.

And even when you do you just ghost and go to another server so you can roll for antag there.

I don't know why your opinion matters here.

**Comment by AdamElTablawy** [comment]
@Wilchenx i do neither of those things first off, and aside from that throwing shade at me doesn't actually do anything to either make what i say less correct nor make the PR less stupid

i'd know, i've done the same to no effect

**Pull request merged**